scripts/post-build.js

```diff
@@ -11,10 +11,12 @@ module.exports = function(fs, path, log, options, done) {
     function(callback) {
       fs.readFile(path.join(pathToCourseFolder, 'config.json'), function(err, data) {
         if (err) {
+          log('Error: ', err);
           return callback(err);
         }
 
         configJson = JSON.parse(data.toString());
+        log('configJson: ', configJson);
```

Comment:
Are these logs supposed to be included or just hangover from a debugging check?

Reply:
The logs were kept intentionally as they could be helpful for debugging over the LMS, but there's no harm in removing them if needed.

Reply:
The logs have been removed, and a minor adjustment has been made to the endpoint URL to include a trailing slash, as some LMSs incorrectly interpret it without the slash.